Dockerfile

```diff
@@ -56,8 +56,7 @@ ENV JDBC_URL='' \
 ENV JDBC_DRIVER_URI=''
 
 # Provide variables for the JDBC connection string
-ENV JDBC_MIN_ACTIVE=50 \
-    JDBC_MAX_ACTIVE=250 \
+ENV JDBC_MAX_ACTIVE=250 \
```

Comment:
It's possible we may want to update the environment variable names to match the new setting names, but I'm not sure if it's worth the effort that would be needed to keep it compatible with the old ones.

Reply:
Lets leave for now if only to keep with backwards compatibility otherwise we will be in whole other world of mess we don't want to deal with.